contrib/http_util/src/main/java/io/opencensus/contrib/http/AbstractHttpHandler.java

```diff
@@ -66,36 +69,34 @@ private static void putAttributeIfNotEmptyOrNull(Span span, String key, @Nullabl
   }
 
   /**
-   * Instrument an HTTP span after a message is sent.
+   * Instrument an HTTP span after a message is sent. Typically called when last chunk of request or
+   * response is sent.
    *
-   * @param span the span.
-   * @param messageId an id for the message.
-   * @param messageSize the size of the message.
+   * @param context request specific {@link HttpRequestContext}
    * @since 0.18
```

Comment:
Please send me the PR to fix all since labels as I commented before.

Reply:
done.
https://github.com/census-instrumentation/opencensus-java/pull/1636